src/Microsoft.VisualStudio.Services.Agent/AgentService.cs

```diff
@@ -19,8 +19,8 @@ public interface IAgentService
 
     public abstract class AgentService
     {
-        protected IHostContext HostContext { get; private set; }
```

Comment:
i did not realize HostContext is not used for anything in SecretMasker. I'll leave it null and will revert the change in AgentService (have private set again)